.travis.yml

```diff
@@ -8,7 +8,7 @@ env:
     - SCVERSION="stable"
 
 before_install:
-    - sudo apt-get install -y xz-utils
+    - sudo apt-get install -y xz-utils libgirepository1.0-dev gcc libcairo2-dev pkg-config python3-dev gir1.2-gtk-3.0
```

Comment:
I believe that this does in fact need libcairo2-dev but I'm curious what could be pulling it in.

Reply:
Good question. I didn't dive in to find out though the dependencies came from https://pygobject.readthedocs.io/en/latest/getting_started.html#ubuntu-getting-started